react/features/toolbox/components/web/Toolbox.js

```diff
@@ -983,6 +999,13 @@ class Toolbox extends Component<Props> {
                 && <SharedVideoButton
                     key = 'sharedvideo'
                     showLabel = { true } />,
+            this._shouldShowButton('shareaudio') && isScreenAudioSupported()
+                && <OverflowMenuItem
+                    accessibilityLabel = { t('toolbar.accessibilityLabel.shareaudio') }
+                    icon = { IconShareAudio }
+                    key = 'shareaudio'
```

Comment:
I think the button name "Share audio" is little confusing. Do you think `Start Audio Share` and `Stop Audio Share` make more sense here ?

Reply:
As it stands the button doesn't have a state it's just Share audio for both stop and start. We're planning on adding a "How to" dialog when the user clicks that button and add that state then. This is meant to be an initial iteration. But if you think it's something we must have for this PR, I can add that state now.